src/components/AssetAccounts/AssetAccountRow.tsx

```diff
@@ -54,10 +54,10 @@ export const AssetAccountRow = ({
 
   const filter = useMemo(() => ({ assetId: rowAssetId, accountId }), [rowAssetId, accountId])
 
-  const fiatBalance = useAppSelector(s => selectFiatBalanceIncludingStakingByFilter(s, filter))
   const cryptoHumanBalance = useAppSelector(s =>
-    selectCryptoHumanBalanceIncludingStakingByFilter(s, filter),
+    selectPortfolioCryptoPrecisionBalanceByFilter(s, filter),
   )
+  const fiatBalance = useAppSelector(s => selectPortfolioFiatBalanceByFilter(s, filter))
```

Comment:
is this the same now that it doesn't include staking?

Reply:
Yeah this file looks like it shouldn't be diffed? Also, while getting the LSP usages of this, looks like `src/components/AssetAccounts/AssetAccounts.tsx` isn't used anymore which we could remove (in a follow-up PR, doesn't have to be in this one)